hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/HapiSuite.java

```diff
@@ -227,19 +235,20 @@ private FinalOutcome runSuite(final Consumer<List<HapiSpec>> runner) {
         }
 
         List<HapiSpec> specs = getSpecsInSuite();
+        final var name = name();
         for (final var spec : specs) {
             if (!overrides.isEmpty()) {
                 spec.addOverrideProperties(overrides);
             }
+            spec.setSuitePrefix(name);
+            spec.setResultLogger(getResultsLogger());
```

Comment:
One little `s` makes all the difference.  Too bad the naming is easily confused.  Unless you can fix it?

Reply:
I can fix the naming, I'll do that for the next update.